WordPress/src/main/java/org/wordpress/android/ui/FilteredRecyclerView.java

```diff
@@ -56,19 +62,21 @@
     private int mSpinnerDrawableRight;
     private AppLog.T mTAG;
 
+    private boolean mToolbarVisibilityLock = false;
```

Comment:
Just out of curiosity, I'm wondering why these vars are being initialized to what would already be their default values?

Reply:
It only gets initialized to its default value if the user has defined this class in XML, but if the class is instantiated programmatically, then no attributes would be passed and therefore no default 🤓 But it's good that you mentioned it because I just pulled it up to verify and realized that I had set the default value in attrs to be `true` - that wouldn't have worked well.